plugins/org.jboss.tools.docker.reddeer/src/org/jboss/tools/docker/reddeer/core/ui/wizards/RunADockerImagePageOneWizard.java

```diff
@@ -39,8 +41,12 @@ public RunADockerImagePageOneWizard() {
 	}
 
 	public void finish() {
-		new FinishButton().click();
-		new WaitWhile(new JobIsRunning(), TimePeriod.VERY_LONG);
```

Comment:
This comment is to a wizard class. Please consider renaming wizard to ImageRunSelectionPage (same as class of the wizard page org.eclipse.linuxtools.internal.docker.ui.wizards.ImageRunSelectionPage, easy to look up...) or DockerContainerSettingsWizardPage (this follows page name in header of the wizard page). Choice is yours.